README.md

```diff
@@ -403,8 +405,12 @@ Error: No such file or directory @ rb_sysopen - C:/ProgramData/PuppetLabs/puppet
 Error: Try 'puppet help module install' for usage
   ~~~
 
-For Puppet 4.2.2+ (and 3.8.2) we've decreased the possibility of the issue occurring based on the fixes in [PUP-4854](https://tickets.puppetlabs.com/browse/PUP-4854). A complete fix will become available in a future version of Puppet that incorporates [PUP-4866](https://tickets.puppetlabs.com/browse/PUP-4866).
-If you are affected by this issue, you can work around this by downloading the `.tar.gz` from the Forge and use `puppet module install` using the file rather than directly from the Forge.
+  For Puppet 4.2.2+ (and 3.8.2) we've decreased the possibility of the issue occurring based on the fixes in [PUP-4854](https://tickets.puppetlabs.com/browse/PUP-4854). A complete fix will become available in a future version of Puppet that incorporates [PUP-4866](https://tickets.puppetlabs.com/browse/PUP-4866).
+  
+  If you are affected by this issue, you can work around this by;
+  - Use the `--module_working_dir` parameter to set a different temporary directory which has a smaller length e.g.
```

Comment:
instead of "e.g" use "for example". This helps our users whose first language is not english.